upgrade-2.x.sh

```diff
@@ -387,8 +388,11 @@ function in_container_hostapp_update {
             sleep 10
         fi
     done
-    mkfifo /tmp/resinos-image.docker
-    ${DOCKER_CMD} save "${update_package}" > /tmp/resinos-image.docker &
+
+    tmp_image=$(mktemp -u)
```

Comment:
Would be nice to have a prefix here to namespace these files.

Reply:
@agherzan added, see the latest version of the PR. Will add another PR for namespacing the other case where we use `mktemp` as well.

Reply:
Looks good.